Makefile

```diff
@@ -36,10 +36,20 @@ TESTFLAGS    :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# Static linking with glibc is a bad time; see
+# https://github.com/golang/go/issues/13470.
+# If a static build is requested, assume musl is installed (it is in
+# the cockroachdb/builder docker container) and link against it
+# instead.
+CC = /usr/local/musl/bin/musl-gcc
```

Comment:
Seems like fully-static builds was an experiment which failed. Why not just remove support for it completely?